backend/api/models/FuelCode.py

```diff
@@ -0,0 +1,126 @@
+"""
+    REST API Documentation for the NRS TFRS Credit Trading Application
+
+    The Transportation Fuels Reporting System is being designed to streamline
+    compliance reporting for transportation fuel suppliers in accordance with
+    the Renewable & Low Carbon Fuel Requirements Regulation.
+
+    OpenAPI spec version: v1
+
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+from django.db import models
+from auditable.models import Auditable
+
+from api.models.FuelCodeStatus import FuelCodeStatus
+
+
+class FuelCode(Auditable):
+    """
+    Fuel codes for the Secure Document Upload
+    """
+    fuel_code = models.CharField(
```

Comment:
Should this be a lookup field, or is this the latter part of a fuel code being BCLCF101.4, BCLCF488.11, etc.? Would this column contain the "BCLCF" pre-fix? In very early iterations of fuel codes, years ago, we had talked about having the BCLCF as a discreet attribute... which would allow for other types in the future.

Reply:
Right now it is a free-form text with the idea of BCLCF being the prefix. 

To allow more types in the future, I'll just automatically insert BCLCF as part of the string, instead of assuming it'll always be BCLCF